docs/digger/installation/nagios-digger-health-check.asciidoc

```diff
@@ -1,22 +1,22 @@
-== Verify The Installation
+== Nagios Installation Health checks
 
-To monitor the status of the Digger components, the Digger installer will also install https://www.nagios.org/[Nagios^] and several Nagios checks that monitor. At the end of the installation Ansible will trigger each of the checks and output the status of these checks. To navigate to Nagios and manually verify that all checks are passing, follow the steps below.
+To monitor the status of the Digger components, the Digger installer will also install https://www.nagios.org/[Nagios^] and several Nagios checks that monitor. At the end of the installation Ansible will trigger each of the checks and output the results of these checks. 
```

Comment:
shoul remove the end of this at end of sentence `checks that monitor`. It's duplicated

Reply:
resolved